modules/custom/social_demo/social_demo.drush.inc

```diff
@@ -5,72 +5,72 @@
  * Contains social_demo.drush.inc.
  */
 
+use Drush\Log\LogLevel;
+use Drupal\user\Entity\User;
+
 /**
  * Implements hook_drush_command().
  */
 function social_demo_drush_command() {
-  $items['social-demo-add'] = array(
```

Comment:
Is there any specific reason why this got changed? This means all of our scripts that run demo content will now not work anymore.. would be nice to make it backwards compatible in the future :)